src/packageManager/packageManager.c

```diff
@@ -164,17 +163,22 @@ void loadRessources()
             char cShort[MAX_LAYER_NAME_SIZE];
             strcpy(cShort, ep->d_name);
 
-            size_t len = strlen(ep->d_name);
+            const char *file_name = ep->d_name;
+            if (strcmp(file_name, ".") == 0 || strcmp(file_name, "..") == 0)
```

Comment:
Why did you add these? It's not necessary when we have line 173.

Reply:
Agree